lambdas/service/lambda-policy.json.template.py

```diff
@@ -122,6 +122,15 @@
                 f"arn:aws:states:{aws.region_name}:{aws.account}:execution:{config.cart_item_state_machine_name}:*",
                 f"arn:aws:states:{aws.region_name}:{aws.account}:execution:{config.cart_export_state_machine_name}"
             ]
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "apigateway:GET"
+            ],
+            "Resource": [
+                f"arn:aws:apigateway:{aws.region_name}::/restapis/{aws.api_gateway_id('service')}/stages/{config.deployment_stage}/exports/oas30"
```

Comment:
Line longer than 120 char, however I'm not certain the "fix" below makes things better. Possibly?
```suggestion
                f"arn:aws:apigateway:{aws.region_name}::"
                f"/restapis/{aws.api_gateway_id('service')}/stages/{config.deployment_stage}/exports/oas30"
```

Reply:
I'll leave it and see if Hannes complains.

Reply:
I would wrap it there, too.

Reply:
Changed.